### PR TITLE
Fix handlers to use different class as a key in mapsubtype class

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java [hunk 1]
```diff
@@ -80,6 +80,7 @@ protected void releaseLiquibaseLock(DatabaseConfig config)
       if (!existsStatus) {
         LOGGER.info("Table database_change_log_lock does not exists in DB");
         LOGGER.info("Proceeding with liquibase assuming it has never been run");
+        return;
```
Comment (conradoverta) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java:83]: This seems unrelated, no?

**Review by conradoverta — APPROVED** [review]
Nice

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java:83]: Yes but since it is just one line and related to jdbi changes so i thought it will works here. :)
